scripts/publish-release.js

```diff
@@ -76,14 +76,15 @@ const cwd = process.cwd()
           setTimeout(resolve, retryDelaySeconds * 1000)
         )
         await publish(pkg, retry + 1)
+      } else {
+        throw err
       }
-      throw err
     } finally {
       publishSema.release()
     }
   }
 
-  await Promise.all(
+  await Promise.allSettled(
```

Comment:
Is the idea here that we don't want to error if one of the packages doesn't publish?

Perhaps this should print the `status` so we know which ones rejected?

Reply:
The idea is that it should finish all packages even if one of them failed.

Reply:
Doe this relate to the jump to 13.5.0 release and related canary's?